install/kubernetes/cilium/templates/_hubble-generate-certs-job-spec.tpl

```diff
@@ -10,7 +10,7 @@ spec:
       serviceAccountName: {{ .Values.serviceAccounts.hubblecertgen.name | quote }}
       containers:
         - name: certgen
-          image: {{ .Values.certgen.image.repository }}:{{ .Values.certgen.image.tag }}
+          image: {{ template "container.image" (dict "registry" $.Values.registry "image" .Values.certgen.image) }}
```

Comment:
Why is the `$` sign needed here? I thought it was only needed inside `with` or `range` etc (scopes that redefine `.`)? The `.Values.certgen.image` doesn't have it either?

Reply:
Right I've check and they're not needed. I thought some instances were scoped but after re-reading we don't have any. I'll remove all the `$`.